components/button/style.scss

```diff
@@ -26,7 +26,8 @@
 		box-shadow: inset 0 -1px 0 #cccccc;
 		vertical-align: top;
 
-		&:hover {
+		&:hover,
+		&.components-icon-button:not(:disabled):not([aria-disabled="true"]):hover {
```

Comment:
I feel it should be the opposite: avoid overriding the hover styles in `components-icon-button`'s stylesheet if the button has the `is-default` className.

Reply:
So add something like a `:not( .is-default )` there?

Reply:
yes, It's equivalent right? My idea is that the `IconButton` component uses the `Button` component which means it knows about it's existence but not the opposite

Reply:
Should be fixed now!